src/context/AppContext.js

```diff
@@ -113,6 +115,18 @@ function getRoutes(user) {
         !user ||
         user.policy.organization !== undefined,
     },
+    {
```

Comment:
We have specific policies for stakeholder management now (or at least we will when https://github.com/Greenstand/treetracker-database/pull/92 is merged): `manage_stakeholders` and `list_stakeholders`

Reply:
I added those policies with the current implementation but I think we'll have to revisit after the new policies are merged.